app/src/main/res/values/strings.xml

```diff
@@ -1116,5 +1116,6 @@
     <string name="no_data_fetched">No Data Fetched</string>
     <string name="recorder">Recorder</string>
     <string name="save_graph">Save Graph</string>
-
+    <string name="import_log_action">Import Log</string>
+    <string name="import_data_log_no_selection_error">Kindly select one of the devices</string>
```

Comment:
We don't add string resources at the end of this file. It will make a lot of merge conflicts. Please move them up where similar text can be found.